tests/default_batch.spec.ts

```diff
@@ -18,7 +18,7 @@ test("Sets default batch for today", async ({ page }) => {
   await when_i_press_confirm();
   await then_i_should_see_a_success_message();
 
-  await when_i_click_on_the_tab("Action needed");
+  await when_i_click_on_the_tab("Action needed (7)");
```

Comment:
Wonder if it would be worth changing these tests to not need the count.

Reply:
Yes... 
One **really** annoying thing is that the tab IDs also change if the label changes. I'm also not sure that the `c.with_tab` helper allows overriding the `id` attribute.